packages/collections/src/list/mutable/ArrayList.ts

```diff
@@ -1,12 +1,14 @@
 import { ReadOnlyCollection } from '../../collection/readonly/ReadOnlyCollection';
-import { Cloneable } from '../../../base/Cloneable';
-import { EqualsFunction } from '../../../comparison/equality/EqualsFunction';
-import { StrictEquals } from '../../../comparison/equality/StrictEquals';
+import {
+  Cloneable,
+  EqualsFunction,
+  StrictEquals,
+  IndexOutOfBoundsException,
```

Comment:
I propose to move `IndexOutOfBounds` to `collections` package, closer to usage scope.

Reply:
Moved to "exceptions"scope":ok_hand:
Please take a look